src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -47,6 +48,17 @@ public class NodeAutoCompleteSearchViewModel : SearchViewModel
         private const string nodeClusterAutocompleteMLEndpoint = "MLNodeClusterAutocomplete";
         internal bool IsOpen { get; set; }
 
+        private bool resultsLoaded;
+        public bool ResultsLoaded
```

Comment:
Looks like we need this to be public due to the binding. Let's add a comment an d also make it obsolete.